File: 02 백준/08 정렬/Python/Prob15688.py

"""
title  : 15688. 수 정렬하기 5 (Silver 5)
time   : 20664ms      12192ms
memory : 83812KB      137356KB
"""

import sys

def solution():
    input = sys.stdin.readline

    N = int(input())
    numbers = [int(input()) for _ in range(N)]

    for num in sorted(numbers):
        print(num)

solution()

File: 02 백준/08 정렬/Java/src/Prob15688.java

/**
* title  : 15688. 수 정렬하기 5 (Silver 5)
* time   : 20568ms
* memory : 108028KB
*/

import java.io.*;
import java.util.*;

public class Prob15688 {
    
    public static void main(String[] args) throws IOException {

        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringBuilder sb = new StringBuilder();

        int N = Integer.parseInt(br.readLine());

        int[] number = new int[N];
        for (int i = 0; i < N; i++) {
            number[i] = Integer.parseInt(br.readLine());
        }

        Arrays.sort(number);

        for(int i = 0; i < N; i++) {
            sb.append(number[i]).append("\n");
        }

        System.out.println(sb);
    }
}
